### ISSUE TITLE
Brim not created around small nested objects

### ISSUE THREAD

**Issue by Tinchus2009** [alexrj/Slic3r#4098]
### Version
latest dev and previous ones too

### Operating system type + version
win10 64b

### Behavior
 If you have a full palte and need space sometimes you arrange small objects inside biger ones. The problem now arises because in you use a brim, those smaller objeect dont get one.

In the picture you can see an example

![brim](https://user-images.githubusercontent.com/14299602/29743319-fd8c6d3e-8a65-11e7-91d4-306258f9b2e7.png)

**Comment by lordofhyphens** [comment]
Have you set an interior brim?

Also, could you submit an AMF and config you see this problem with?

**Comment by Tinchus2009** [comment]
Yes, I tried with interior brim, wich makes a brim in the interior of all objects, included the small nested one, but still thoses nested objects dont get an exterior brim. Config file and amf file atached
[falla_brim.zip](https://github.com/alexrj/Slic3r/files/1254105/falla_brim.zip)

**Comment by lordofhyphens** [comment]
My suspicion is that the nested parts are being masked by the one they are inside, because the outer brim logic only looks at the outer contours when making brims. The holes are available too.

One of the things we could do is make brims symmetrical (I suspect this should have been the case in the first place), but that tends to generate brims everywhere (including small holes).

**Comment by lordofhyphens** [comment]
![image](https://user-images.githubusercontent.com/31754/30002625-9c6d4dfc-9073-11e7-87cc-002f142c36e9.png)
Because that's what happens if I make the brim symmetrical immediately. The interior brim area becomes the max of the interior brim configuration and the exterior.

**Comment by lordofhyphens** [comment]
[4098.zip](https://github.com/alexrj/Slic3r/files/1272916/4098.zip)
Here's my test case, simplified slightly, from the original plate.

Ideally, with the provided settings there should be no brim generated inside the holes for the same part and nothing is overlapping.

**Comment by Tinchus2009** [comment]
I downloaded your zip. Then loaded the amf file, then loaded your config.ini, the brim is still not being generated for the nsted object
